siriuspy/siriuspy/devices/blm.py

```diff
@@ -0,0 +1,57 @@
+"""Beam Loss Monitor devices."""
+
+from ..search import BLMSearch as _BLMSearch
+from .device import Device as _Device
+
+
+class BLM(_Device):
+    """Beam Loss Monitor Device."""
+
+    class DEVICES:
+        """Devices names."""
+        SENSOR1 = _BLMSearch.conv_blmname_2_devname('SENSOR1')
```

Comment:
Careful here, because if the information regarding BLMs move to the control-system-constants repository in the future, just like the other Search classes infos, this code will communicate with the constants server in import time. We do not do this.

Try to use class properties instead, like [here](https://github.com/lnls-sirius/dev-packages/blob/5d27f2ac8a2bd4fbde0e157d290de111a2e6766f/siriuspy/siriuspy/timesys/csdev.py#L90). This way the code will be robust against this possible change.

Reply:
well pointed. implemented.